packages/alfa-rules/src/index.ts

```diff
@@ -82,9 +82,7 @@ export namespace Flattened {
  *
  * @public
  */
-export const Flattened: Flattened.Flattened = Sequence.from<Flattened.Rule>(
-  Rules.values()
-);
+export const Flattened = Sequence.from(Rules.values()) as Flattened.Flattened;
```

Comment:
This reverts a bit what was done in #1197 because that version causes the build to break on non-clean state, requiring to `clean` and re-build the full `alfa-rules` package, which is a annoyance 🙃